src/development/ui/advanced/splash-screen.md

```diff
@@ -39,11 +45,21 @@ part of the [Human Interface Guidelines][].
 
 ## Android launch screen
 
+{{site.alert.warning}}
+If you are experiencing a crash from implementing a splash screen, you might
+need to migrate your code. See detailed instructions in the
+[Deprecated Splash Screen API Migration][].
+{{site.alert.warning}}
+
 In Android, there are two separate screens that you can control:
 a _launch screen_ shown while your Android app initializes,
 and a _splash screen_ that displays while the Flutter experience
 initializes.
 
+As of Flutter 2.5, Flutter has consolidated these two screens
+into one Android launch screen. Flutter automatically keeps this
+launch screen displayed until it draws the first frame.
+
```

Comment:
@sfshaza2 Maybe this should be emphasized?

Reply:
I deleted the last paragraph because that isn't really true. You have to just use the launch screen from here on out.

The second paragraph is mentioned in the warning before, did you want to keep both?